lib/shift_values.rb

```diff
@@ -0,0 +1,20 @@
+class ShiftValues
+
+  def squared(date)
+    (date.to_i)**2
+  end
+
+  def key_values(num)
+    num.chars.each_cons(2).map {|numbers| numbers.join.to_i}
```

Comment:
generates key values array
takes a provided number string, breaks it into an array with each digit string 
then selects two consecutive digit strings, joins them, then turns that joined number string into an integer
end up with an array of four  two-digit numbers